pyconnect/clients/kafka.py

```
@@ -300,14 +296,18 @@ def listen(self, topics: List[str], callback: Callable):
 def create_kafka_listeners():
     """
     Create an instance of each Kafka listener.  Add additional listeners as needed.
+    Call the method to start each listener, then add the listener to the list for shutdown.
+
+    Example:
+            listener = start_sync_event_listener()
+            kafka_listeners.append(listener)
```

Removed unused event listeners, but documented the use.  The plan is to leave the listener code in place, as this is a common way to extend LFH.  At this point, we only create the sync topic here, we don't start any listeners.

---

Is the create topic command idempotent (programming word of the day). In other words will it just no-op or fail silently if the topic already exists?

---

it checks to see if the topic exists before trying to create it, so yes, idempotent